Automattic/jetpack
# Premium content: Move view switcher to drop down

Fixes #22745

#### Changes proposed in this Pull Request:

On small (<782px viewport) use a drop-down rather than two separate buttons to switch between guest and subscriber views. This takes up less space allowing mobile users to see the whole menu.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






1. Use a premium or higher wpcom site
2. Create a new post
3. Resize your viewport to mobile size
4. Insert a premium content block
5. Use the new drop down menu to switch between subscriber and guest views
6. When you switch, the content being displayed should change to match
7. Make the viewport bigger
8. Note that the drop-down is replaced by two separate buttons
9. Make the viewport smaller again, without reloading. Note that the menu uses a dropdown on smaller screens and most importantly doesn't mysteriously disappear.

Before | After
-------|------
![image](https://user-images.githubusercontent.com/93301/157327283-73124dfb-c302-4aba-86ee-85d7db763211.png)| ![image](https://user-images.githubusercontent.com/93301/157327336-ee7dce56-77cc-4878-94f5-d46526b78e33.png)
![image](https://user-images.githubusercontent.com/93301/157327427-8567baba-20ba-490e-b7bd-e70e668119cc.png) | ![image](https://user-images.githubusercontent.com/93301/157327487-46d5779e-2e04-4eb0-b6fd-f0b421068491.png)
